js/package.json

```diff
@@ -26,5 +26,8 @@
   },
   "jest": {
     "preset": "ts-jest"
+  },
+  "dependencies": {
```

Comment:
is this really a new dependency? this wasn't here before the last commit fixing typos

Reply:
removed